configure.ac

```diff
@@ -67,6 +67,11 @@ AC_ARG_WITH([bashcompletiondir],
        [BASHCOMPLETIONDIR=$withval], [BASHCOMPLETIONDIR="${datadir}/bash-completion/completions"])
 AC_SUBST(BASHCOMPLETIONDIR)
 
+AC_ARG_WITH([ifcfgdir],
+       AS_HELP_STRING([--with-ifcfgdir=DIR], [The ifcfg base configuration directory]),
+       [IFCFGDIR=$withval], [IFCFGDIR="network-scripts"])
```

Comment:
This should be a full path configurable. We should not assume it's relative to /etc/sysconfig.